feature-libs/my-account/organization/occ/converters/occ-org-unit-address-list-normalizer.ts

```diff
@@ -1,27 +1,27 @@
 import { Injectable } from '@angular/core';
-import { B2B_ADDRESS_NORMALIZER } from '@spartacus/my-account/organization/core';
 import {
+  Address,
+  ADDRESS_NORMALIZER,
   Converter,
-  Occ,
-  EntitiesModel,
-  B2BAddress,
   ConverterService,
+  EntitiesModel,
+  Occ,
 } from '@spartacus/core';
 
 @Injectable()
 export class OccOrgUnitAddressListNormalizer
```

Comment:
This could be renamed to `OccAddressListNormalizer` and moved to the core and registered there.